earth_enterprise/rpms/opengee-fusion/pre-install.sh

```diff
@@ -14,31 +14,26 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# NOTE: requires xmllint from libxml2-utils
-
 set +x
 
 #-----------------------------------------------------------------
-# Versions and user names:
+# Definitions
 GEE="Google Earth Enterprise"
 #-----------------------------------------------------------------
 
-# config values
-ASSET_ROOT_VOLUME_SIZE=0
-
-
 #-----------------------------------------------------------------
 # Main Functions
 #-----------------------------------------------------------------
 main_preinstall()
 {
-    service gefusion stop
+    if [ -f /etc/init.d/gefusion ]; then
+        service gefusion stop
+    fi
 
     load_systemrc_config
```

Comment:
I think all install scripts are designed to read `systemrc` before using the values that are stored there.  This seems to allow system administrators to use custom user and group names.

I don't think it's possible to store names of users and groups in a CPIO archive, so that can't be done from Gradle.

The usual practice is to allocate UIDs and GIDs for system users and groups in each distribution.  We could submit a request for `gefusionuser` and `gegroup`.  But, the need for the other users and groups would be eliminated if we unbundled the software components they are associated with.

On the other hand, I think it's a bad idea idea to customize user and group names.  Actually, the Fedora packaging guide recommends that a package never delete user and group names, even if it created them on installation.  User and group removal is left as a manual step for administrators.  That's because there's no good way to ensure there are no files or directories that were created with these UIDs and GIDs that may be used again on that host.